bioimageio/spec/shared/common.py

```diff
@@ -33,9 +33,34 @@ def dump(self, data, stream=None, *, transform=None):
     yaml = MyYAML(typ="safe")
 
 
+try:
+    from tqdm import tqdm  # not available in pyodide
+except ImportError:
+
+    class tqdm:  # type: ignore
+        """no-op tqdm"""
+
+        def __init__(self, iterable: Optional[Iterable] = None, *args, **kwargs):
+            self.iterable = iterable
+
+        def __iter__(self):
+            if self.iterable is not None:
+                yield from self.iterable
+
+        def update(self, *args, **kwargs):
+            pass
+
+
 BIOIMAGEIO_CACHE_PATH = pathlib.Path(
     os.getenv("BIOIMAGEIO_CACHE_PATH", pathlib.Path(tempfile.gettempdir()) / "bioimageio_cache")
 )
+BIOIMAGEIO_USE_CACHE = os.getenv("BIOIMAGEIO_USE_CACHE", "true").lower() in ("false", "no")
```

Comment:
```suggestion
BIOIMAGEIO_USE_CACHE = os.getenv("BIOIMAGEIO_USE_CACHE", "true").lower() not in ("false", "no", "0")
```

Reply:
You need the "not" here so that this evaluates correctly: to make this clear the code **without** not would read like this for the default value
```python
use_cache = "true" in ("false", "no")
# use cache evaluates to False in this case, which is wrong!
```

And I would add 0 here since it's very common to specify env variables by integers.

Reply:
good catch! but then let's get rid of the double negative here, too 🚀 
```suggestion
BIOIMAGEIO_USE_CACHE = os.getenv("BIOIMAGEIO_USE_CACHE", "true").lower() in ("true", "yes", "1")
```

Reply:
or do we want `BIOIMAGEIO_USE_CACHE='ja'` also to work?

Reply:
You're suggestion looks good. Adding more synonyms is fine with me.